smac/facade/smac_bo_facade.py

```diff
@@ -24,6 +24,16 @@ class SMAC4BO(SMAC4AC):
     "Towards Assessing the Impact of Bayesian Optimization's Own Hyperparameters" by Lindauer et al., presented at the
     DSO workshop 2019 (https://arxiv.org/abs/1908.06674).
 
+    Changes are:
+    * Instead of having an initial design of size 10*D as suggested by Jones et al. 1998 (actually, they suggested
+      10*D+1), we use an initial design of 8*D.
```

Comment:
Could you also add why we use 8D?

Reply:
Because https://arxiv.org/abs/1908.06674 says so. I was hoping that's clear from the above comment.

Reply:
Yes, sure.